FirebaseDatabase/CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
 # Unreleased
 - [added] Made emulator connection API consistent between Auth, Database, Firestore, and Functions (#5916).
+- [added] Implement RTDB Query get (#7110)
```

Comment:
These release notes should spark users interest, explain what we did and how the user can benefit from it. As an example, I submitted this for Android: https://critique-ng.corp.google.com/cl/345732711